src/domain/collaboration/callout/CalloutView/CalloutView.tsx

```
@@ -19,6 +19,10 @@ import { type LocationStateCachedCallout, LocationStateKeyCachedCallout } from '
 import CalloutContributionsBlock from '../../calloutContributions/CalloutContributionsBlock';
 import CalloutContributionsHorizontalPager from '../../calloutContributions/CalloutContributionsHorizontalPager';
 import CalloutContributionPreview from '../../calloutContributions/calloutContributionPreview/CalloutContributionPreview';
+import CalloutContributionDialogCollaboraDocument from '../../calloutContributions/collaboraDocument/CalloutContributionDialogCollaboraDocument';
+import CalloutContributionPreviewCollaboraDocument from '../../calloutContributions/collaboraDocument/CalloutContributionPreviewCollaboraDocument';
```

Biome (2.4.10)

[error] This import is unused.

(lint/correctness/noUnusedImports)